Description: Replace /usr/bin/bash with /bin/bash in shebangs
 bash is located in /usr/bin on Arch Linux, which is the distribution
 that this program was intended for. On Debian, it's located in /bin,
 so the shebangs must be changed to make the scripts executable.
Author: Ben Westover <me@benthetechguy.net>
Forwarded: not-needed
Last-Update: 2022-09-16

--- a/scripts/libmakepkg/buildenv/buildflags.sh.in
+++ b/scripts/libmakepkg/buildenv/buildflags.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   buildflags.sh - Clear user-specified buildflags if requested
 #
--- a/scripts/libmakepkg/buildenv/compiler.sh.in
+++ b/scripts/libmakepkg/buildenv/compiler.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   compiler.sh - CCache and DistCC compilation
 #   ccache - Cache compilations and reuse them to save time on repetitions
--- a/scripts/libmakepkg/buildenv/debugflags.sh.in
+++ b/scripts/libmakepkg/buildenv/debugflags.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   debugflags.sh - Specify flags for building a package with debugging
 #   symbols
--- a/scripts/libmakepkg/buildenv/lto.sh.in
+++ b/scripts/libmakepkg/buildenv/lto.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   lto.sh - Specify flags for building a package with link-time
 #            optimisation
--- a/scripts/libmakepkg/buildenv/makeflags.sh.in
+++ b/scripts/libmakepkg/buildenv/makeflags.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   makeflags.sh - Clear user-specified makeflags if requested
 #
--- a/scripts/libmakepkg/executable/ccache.sh.in
+++ b/scripts/libmakepkg/executable/ccache.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   ccache.sh - Confirm presence of ccache binary
 #
--- a/scripts/libmakepkg/executable/checksum.sh.in
+++ b/scripts/libmakepkg/executable/checksum.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   checksum.sh - Confirm presence of binaries for checksum operations
 #
--- a/scripts/libmakepkg/executable/distcc.sh.in
+++ b/scripts/libmakepkg/executable/distcc.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   distcc.sh - Confirm presence of distcc binary
 #
--- a/scripts/libmakepkg/executable/fakeroot.sh.in
+++ b/scripts/libmakepkg/executable/fakeroot.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   fakeroot.sh - Confirm presence of fakeroot binary
 #
--- a/scripts/libmakepkg/executable/gpg.sh.in
+++ b/scripts/libmakepkg/executable/gpg.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   gpg.sh - Confirm presence of gpg binary
 #
--- a/scripts/libmakepkg/executable/gzip.sh.in
+++ b/scripts/libmakepkg/executable/gzip.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   gzip.sh - Confirm presence of gzip binary
 #
--- a/scripts/libmakepkg/executable/pacman.sh.in
+++ b/scripts/libmakepkg/executable/pacman.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   pacman.sh - Confirm presence of pacman binary
 #
--- a/scripts/libmakepkg/executable/strip.sh.in
+++ b/scripts/libmakepkg/executable/strip.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   strip.sh - Confirm presence of strip binary
 #
--- a/scripts/libmakepkg/executable/sudo.sh.in
+++ b/scripts/libmakepkg/executable/sudo.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   sudo.sh - Confirm presence of sudo binary
 #
--- a/scripts/libmakepkg/executable/vcs.sh.in
+++ b/scripts/libmakepkg/executable/vcs.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   vcs.sh - Confirm presence of binaries for VCS operations
 #
--- a/scripts/libmakepkg/lint_config.sh.in
+++ b/scripts/libmakepkg/lint_config.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   lint_config.sh - functions for checking for makepkg.conf errors
 #
--- a/scripts/libmakepkg/util/dirsize.sh.in
+++ b/scripts/libmakepkg/util/dirsize.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   dirsize.sh - calculate size of all files in a directory
 #
--- a/scripts/libmakepkg/executable/debugedit.sh.in
+++ b/scripts/libmakepkg/executable/debugedit.sh.in
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
 #
 #   debugedit.sh - Confirm presence of debugedit binary
 #
